tests/test_conversion.py

```diff
@@ -0,0 +1,48 @@
+import sinabs
+import sinabs.layers as sl
+import torch.nn as nn
+
+
+def test_layer_replacement_sequential():
+    model = nn.Sequential(
+        nn.Conv2d(2, 8, 3),
+        sl.IAF(spike_threshold=2.0),
+        nn.Conv2d(8, 16, 3),
+        sl.IAF(spike_threshold=1.0),
+        nn.Flatten(),
+        nn.Linear(512, 10),
+    )
+
+    mapper_fn = lambda module: sl.IAFSqueeze(**module._param_dict, batch_size=18)
```

Comment:
I suggest adding a property (getter) or a get-function to the `StatefulLayer` class, to allow for safe access of the `_param_dict` without leading underscore. 

We might also use this as an opportunity to introduce a better name here. The term `param` could be misinterpreted as referring to paramters in the pytorch sense.

Reply:
I'm not sure what you mean by your first paragraph, maybe we can have a chat about this. For clarification, StatefulLayer already has a _param_dict property 
https://github.com/synsense/sinabs/blob/develop/sinabs/layers/stateful_layer.py#L143